<!-- MHonArc v2.6.16 -->
<!--X-Subject: Re: [bug #21481] System test fail with "ImportError: No module named	dauvergne_protocol" -->
<!--X-From-R13: whfgva <wyrpNtragbb.bet> -->
<!--X-Date: Tue, 14 Jan 2014 12:02:20 +0100 -->
<!--X-Message-Id: 52D51918.5070303@gentoo.org -->
<!--X-Content-Type: multipart/signed -->
<!--X-Reference: 20140114&#45;104023.sv21202.67165@gna.org -->
<!--X-Reference: 20140114&#45;104204.sv21202.20066@gna.org -->
<!--X-Reference: CAED9pY8peLdrpHHx1BzedrX58ar52Gn7rwEGj7F5+pVonVPE&#45;g@mail.gmail.com -->
<!--X-Derived: pgp8dvRMgK5Nd.pgp -->
<!--X-Head-End-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
   "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Re: [bug #21481] System test fail with &quot;ImportError: No module named	dauvergne_protocol&quot; -- January 14, 2014 - 12:02</title>
<link rel="stylesheet" type="text/css" href="/mail.gna.org/archives-color-gna.css"> 
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
</head>
<body>
<!--X-Body-Begin-->
<!--X-User-Header-->
<!--X-User-Header-End-->
<!--X-TopPNI-->
<h2><img src="/mail.gna.org/images/mail.orig.png" width="48" height="48"
alt="mail" class="pageicon" />Re: [bug #21481] System test fail with &quot;ImportError: No module named	dauvergne_protocol&quot;</h2>
<br />
<div class="topmenu">
<a href="../" class="tabs">Others Months</a> | <a href="index.html#00011" class="tabs">Index by Date</a> | <a href="threads.html#00011" class="tabs">Thread Index</a><br />
<span class="smaller">&gt;&gt;&nbsp;&nbsp;
[<a href="msg00010.html">Date Prev</a>] [<a href="msg00012.html">Date Next</a>] [<a href="msg00010.html">Thread Prev</a>] [<a href="msg00012.html">Thread Next</a>]
</div>

<!--X-TopPNI-End-->
<!--X-MsgBody-->
<!--X-Subject-Header-Begin-->
<h3><a name="header" href="#header">Header</a></h3>
<!--X-Subject-Header-End-->
<!--X-Head-of-Message-->
<ul class="headdata">
<li class="menuitem">
<em>To</em>: Edward d'Auvergne &lt;edward@xxxxxxxxxxxxx&gt;</li>
<li class="menuitem">
<em>Date</em>: Tue, 14 Jan 2014 12:01:44 +0100</li>
<li class="menuitem">
<em>Cc</em>: &quot;relax-devel@xxxxxxx&quot; &lt;relax-devel@xxxxxxx&gt;</li>
<li class="menuitem">
<em>Message-id</em>: &lt;<a href="msg00011.html">52D51918.5070303@gentoo.org</a>&gt;</li>
<li class="menuitem">
<em>References</em>: &lt;20140114-104023.sv21202.67165@xxxxxxx&gt;	&lt;20140114-104204.sv21202.20066@xxxxxxx&gt;	&lt;CAED9pY8peLdrpHHx1BzedrX58ar52Gn7rwEGj7F5+pVonVPE-g@xxxxxxxxxxxxxx&gt;</li>
<li class="menuitem">
<em>User-agent</em>: Mozilla/5.0 (X11; Linux x86_64;	rv:24.0) Gecko/20100101 Thunderbird/24.2.0</li>
</ul>
<!--X-Head-of-Message-End-->
<!--X-Head-Body-Sep-Begin-->
</div><!-- end headdata -->
<br />
<h3><a name="content" href="#content">Content</a></h3>
<div class="postedby">Posted by <strong>justin</strong> on January 14, 2014 - 12:02:</div>
<div class="msgdata">
<!--X-Head-Body-Sep-End-->
<!--X-Body-of-Message-->
<pre style="margin: 0em;">On 14/01/14 11:52, Edward d'Auvergne wrote:
</pre><blockquote class="blockquote"><pre style="margin: 0em;">Hi Justin,

Would you be able to give some more information as to how you obtained
the errors?  I cannot see such errors on 32-bit or 64-bit
Mageia/Mandrake Linux systems (or 32-bit and 64-bit MS Windows or
64-bit Mac OS X for that matter).  Such an import failure is usually
because there is a file called 'auto_analysis.py' or a directory
called 'auto_analysis' located in the directory from where the tests
are run which is causing the import of the relax 'auto_analysis'
package to be hidden, i.e. the other file or directory is imported
instead hence the dauvergne_protocol module will not be found in
auto_analysis.  What is your setup?  Are you running the tests from
the base relax directory?  Where and how is relax installed?

Cheers,

Edward


</pre></blockquote><pre style="margin: 0em;">

Hi Edward,

I am on a 64bit Gentoo Linux system and it doesn't matter whether relax
is installed into python's sitedir or if I am directly inside the
downloaded tarball.

And I am using the source package.

On my system I have python 2.7.6 and 3.3.3 installed. The test are run
using python 2.

If I drop into a python shell and execute the same import command it
works. And other imports in the testsuite are working as well.

Currently I am checking on an opensuse system.

I will report back when done,

Justin

</pre><p><strong>Attachment:
<a href="pgp8dvRMgK5Nd.pgp" ><tt>signature.asc</tt></a></strong><br>
<em>Description:</em> OpenPGP digital signature</p>

<!--X-Body-of-Message-End-->
<!--X-MsgBody-End-->
<!--X-Follow-Ups-->
</div><!-- end msgdata -->
<br />
<h3><a name="related" href="#related">Related Messages</a></h3>
<div class="relateddata">
<ul><li><strong>Follow-Ups</strong>:
<ul>
<li><strong><a name="00012" href="msg00012.html">Re: [bug #21481] System test fail with &quot;ImportError: No module named	dauvergne_protocol&quot;</a></strong>
<ul><li><em>From:</em> Edward d'Auvergne</li></ul></li>
</ul></li></ul>
<!--X-Follow-Ups-End-->
<!--X-References-->
<ul><li><strong>References</strong>:
<ul>
<li><strong><a name="00008" href="msg00008.html">[bug #21481] System test fail with &quot;ImportError: No module named	dauvergne_protocol&quot;</a></strong>
<ul><li><em>From:</em> Justin</li></ul></li>
<li><strong><a name="00009" href="msg00009.html">[bug #21481] System test fail with &quot;ImportError: No module named	dauvergne_protocol&quot;</a></strong>
<ul><li><em>From:</em> Justin</li></ul></li>
<li><strong><a name="00010" href="msg00010.html">Re: [bug #21481] System test fail with &quot;ImportError: No module named	dauvergne_protocol&quot;</a></strong>
<ul><li><em>From:</em> Edward d'Auvergne</li></ul></li>
</ul></li></ul>
<!--X-References-End-->
<!--X-BotPNI-->
</div><!-- end relateddata -->
<!-- NoBotLinksApartFromRelatedMessages -->

<!--X-BotPNI-End-->
<!--X-User-Footer-->
<!--X-User-Footer-End-->
<div class="footer"></div><br />
<div class="right">Powered by <a href="http://www.mhonarc.org">MHonArc</a>, Updated Tue Jan 14 12:20:20 2014</div>  
</body>
</html>
